core/src/services/split-view.js

```diff
@@ -0,0 +1,215 @@
+import { LuigiConfig, LuigiElements } from '../core-api';
+import { Navigation } from '../navigation/services/navigation';
+import { Iframe } from '../services';
+import {
+  GenericHelpers,
+  IframeHelpers,
+  RoutingHelpers
+} from '../utilities/helpers';
+
+class SplitViewSvcClass {
+  constructor() {
+    this.splitViewValues;
+    this.storedSplitViewValues;
```

Comment:
Probably an item for follow up task. Are we sure we need to have both `splitViewValues` and `storedSplitViewValues`? I refactored it to use only `splitViewValues` and it seems to work.

Reply:
Correct, initially thought there needs to be a "reset" .. like if you press ESC while dragging, so the old value stays. But since we do not provide such functionality, its ok to combine them.